pro/src/ui-kit/form/SelectAutocomplete/__specs__/SelectAutocomplete.spec.tsx

```diff
@@ -130,6 +130,9 @@ describe('src | ui-kit | form | SelectAutocomplete', () => {
         )
         await userEvent.type(screen.getByRole('textbox'), 'al')
         expect(screen.getAllByRole('option')).toHaveLength(6) // Allier, Alpes, Hautes-Alpes, Alpes-Maritimes, Calvados, Cantal
+
+        await userEvent.type(screen.getByRole('textbox'), 'rhon')
+        expect(screen.getAllByRole('option')).toHaveLength(1) // Bouches-du-Rhône
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏On peut ajouter le use case avec plusieurs mots

```js
await userEvent.type(screen.getByRole('textbox'), 'bouches rhon')
expect(screen.getAllByRole('option')).toHaveLength(1) // Bouches-du-Rhône
```